client/src/elm/Translate.elm

```diff
@@ -9820,8 +9820,8 @@ translationSet trans =
                     }
 
                 Backend.NutritionActivity.Model.NCDA ->
-                    { english = "ECD Scorecard"
-                    , kinyarwanda = Nothing
+                    { english = "Child Scorecard"
```

Comment:
Why ECD changed to Child?

Reply:
@Diama1 Is there an issue for this?

Reply:
No, it is not there

Reply:
We don't do anything unless there are requirements.
Please open and issue and have @jeanette-pelizzon / @adamhstewart approve it.
Then, attach the issue to this PR.